tests/integ/modin/frame/test_transpose.py

```diff
@@ -61,6 +62,42 @@ def test_dataframe_transpose_set_single_index(
         )
 
 
+@pytest.mark.parametrize(
+    "transpose_operation, expected_query_count",
```

Comment:
nit- why parameterize on expected_query_count if both examples are 1, instead of `@sql_count_checker(query_count=1)`?

Reply:
Fixed.

I copied this code from `transpose_and_double_transpose_parameterize`, which has the redundant query count, and modified the `marks` for one of the test cases.